lib/realtime/syn_handler.ex

```diff
@@ -15,12 +15,17 @@ defmodule Realtime.SynHandler do
     Endpoint.local_broadcast(Connect.syn_topic(tenant_id), "ready", %{pid: pid, conn: conn})
   end
 
-  def on_registry_process_updated(PostgresCdcRls, tenant_id, _pid, meta, _reason) do
-    # Update that the CdCRls connection is ready
-    Endpoint.local_broadcast(PostgresCdcRls.syn_topic(tenant_id), "ready", meta)
-  end
+  def on_registry_process_updated(scope, tenant_id, _pid, meta, _reason) do
+    scope = Atom.to_string(scope)
+
+    case scope do
+      "realtime_postgres_cdc_" <> _ ->
```

Comment:
:nitpick: What if we keep this as part of the Syn module you created and not expose the prefix here?

```elixir
@postgres_cdc_scope_prefix  Realtime.Syn.PostgresCdc.scope_prefix

case scope do
      @postgres_cdc_scope_prefix <> _ ->
...
```